src/main/resources/flattening/config/main.conf

```diff
@@ -17,3 +17,39 @@ pio = {
 fraude = {
   include "fraude/env.conf"
 }
+
```

Comment:
Do you think this configurations are necessary for the whole project?

Reply:
I dropped every confs related to drees_echantillon